csf_tz/patches/create_the_stock_entry_type.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+    stock_entry_type_list = [ {"name": "To Company", "purpose": "Material Receipt"},  {"name": "From Company", "purpose": "Material Issue"} ]
+
+    for stock_entry_type_data in stock_entry_type_list:
+        stock_entry_doc = frappe.new_doc('Stock Entry Type')
+        stock_entry_doc.name = stock_entry_type_data["name"]
```

Comment:
it is not good to provide name of the doc directly, name should be generated automatic